web/pages/speakers/[slug].tsx

```diff
@@ -39,16 +41,10 @@ const QUERY = groq`
   }`;
 
 type SpeakerSession = {
-  _id: string;
-  title: string;
-  duration: number;
+  session: Pick<Session, '_id' | '_type' | 'title' | 'duration' | 'slug'>;
   programContainingSession: {
     programStart: string;
-    sessions: {
-      _type: string;
-      duration: number;
-      session?: Pick<Session, '_id' | 'duration'>;
-    }[];
+    sessions: ProgramSession[];
```

Comment:
Nit: This type implies that any non-null `session` within will have a `_type`, which is not the case. The query only fetches an `_id`.